source/config/configFlags.py

```diff
@@ -202,3 +202,20 @@ def _displayStringLabels(self):
 			# document formatting settings panel.
 			ReportCellBorders.COLOR_AND_STYLE: _("Both Colors and Styles"),
 		}
+
+
+class AddonsAutomaticUpdate(DisplayStringStrEnum):
+	NOTIFY = "notify"
+	# UPDATE = "update"
+	DISABLED = "disabled"
+
+	@property
+	def _displayStringLabels(self):
+		return {
+			# Translators: This is a label for the automatic update behaviour for add-ons.
+			# It will notify the user when updates are available.
+			self.NOTIFY: _("Notify (default)"),
+			# self.UPDATE: _("Update Automatically"),
```

Comment:
Translators comment please.

By the way, why hasn't it been captured by the check script?

Reply:
This is commented out